packages/dropbox.rb

```diff
@@ -1,59 +1,30 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Dropbox < Package
-  description 'Dropbox simplifies the way you create, share and collaborate.  Bring your photos, docs, and videos anywhere and keep your files safe.'
+class Dropbox < Autotools
+  description 'Dropbox Integration for Nautilus'
   homepage 'https://www.dropbox.com/'
-  version '48.4.58'
-  license 'Dropbox-TOS'
-  compatibility 'i686,x86_64'
-  case ARCH
-  when 'i686'
-    source_url 'https://clientupdates.dropboxstatic.com/dbx-releng/client/dropbox-lnx.x86-48.4.58.tar.gz'
-    source_sha256 '3affb0ade20e26adf337489c6b780a94ddd8d51712a3bd88be2def38b5689570'
-  when 'x86_64'
-    source_url 'https://clientupdates.dropboxstatic.com/dbx-releng/client/dropbox-lnx.x86_64-48.4.58.tar.gz'
-    source_sha256 'bd948a712e21c36d01ad17e72a5893972dd9411408561c3b806dbca2c32ae9a4'
-  end
-  binary_compression 'tar.xz'
+  version '2024.04.17'
+  license 'GPL3+'
+  compatibility 'aarch64 armv7l x86_64'
+  source_url 'https://github.com/dropbox/nautilus-dropbox'
```

Comment:
OK, so this is confusing.  The package name should be `Nautilus_dropbox`.  Dropbox is a legitimate app we should not just rename like this.

Reply:
EDIT: After a bit of research, I now understand how this should be installed on Linux as explained [here](https://help.dropbox.com/installs/linux-commands#How-to-build-the-Dropbox-installer-for-Linux-from-source).  Disregard my earlier comment.